src/voter/voting_round.rs

```diff
@@ -311,6 +311,27 @@ impl<H, N, E: Environment<H, N>> VotingRound<H, N, E> where
 		self.best_finalized.as_ref()
 	}
 
+	/// Return all imported votes for the round (prevotes and precommits).
```

Comment:
This is unused in this crate and iirc `VotingRound` is not exported so this isn't used by anyone.